doc/user/layouts/shortcodes/self-managed/versions/curl-sample-files-local-install.html

```diff
@@ -19,3 +19,5 @@
 
 These configuration files are for local evaluation/testing purposes only and
 not intended for production use.
+
+To enable disk metrics reporting, configure the `storage.storageClass.name` field to `standard` in `sample-values.yaml`.
```

Comment:
uh... I think we should have more docs than this. I assume this is in the helm values for the operator?

Reply:
Yeah .. if you want to allow people to set it for local kind/minikube ...
See https://materialize.com/docs/self-managed/v25.2/installation/configuration/#storage-parameters

Reply:
Also is there any reason we shouldn't just set this to be the default?

Reply:
>uh... I think we should have more docs than this. 

Ah like an explanation why we need to do this to enable disk metrics reporting? Yeah that makes sense to me.